tests/PCA_test.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd 
+import numpy as np
+from PCA import * 
```

Comment:
Import specific classes/functions from our toolkit files when possible
```suggestion
from PCA import Compute_PCA
```